config/containers/container-networking.md

```diff
@@ -19,7 +19,7 @@ point of view of the container.
 
 ## Published ports
 
-By default, when you create a container, it does not publish any of its ports
+By default, when you create/run a container using `docker create` or `docker run`, it does not publish any of its ports
```

Comment:
Perhaps change the `create/run` to "create or run". Also looks like this line is now quite a bit longer than 80 chars (which is what we try to use as width), so may need some wrapping;

```suggestion
By default, when you create or run a container using `docker create` or `docker run`,
it does not publish any of its ports to the outside world. To make a port available
to services outside of Docker, or to Docker containers which are not connected to
the container's network, use the `--publish` or `-p` flag. This creates a firewall
rule which maps a container port to a port on the Docker host. Here are some examples.
```